webapp/certified/helpers.py

```diff
@@ -0,0 +1,35 @@
+def get_download_url(model, model_details):
+    """
+    Return the appropriate ubuntu models.com/download url for the model
+    :param model: a certifiedmodel resource
+    :param model_details: a certifiedmodeldetails resource
+    :return: appropriate ubuntu.com/download url
+    """
+    platform_category = model.get("category", "").lower()
```

Comment:
I missed this at first. You can get category from model_details and don't need to pass model in.